src/components/kubectl/logs.ts

```diff
@@ -0,0 +1,184 @@
+import * as vscode from 'vscode';
+import { Kubectl } from '../../kubectl';
+import { selectContainerForPod, PodSummary, quickPickKindName } from '../../extension';
+import { isPod } from '../../kuberesources.objectmodel';
+import * as kuberesources from '../../kuberesources';
+import { ResourceNode } from '../../explorer';
+import * as yaml from 'js-yaml';
+import * as kubectlUtils from '../../kubectlUtils';
+import { host } from '../../host';
+
+export enum LogsDisplayMode {
+    Show,
+    Follow
+}
+
+const FOLLOW_LOGS_TERMINAL = "Kubernetes: Follow Logs";
+
+/**
+ * Fetches logs for a Pod. Handles use cases for fetching pods
+ * from an open document, or from the current namespace.
+ * @param kubectl
```

Comment:
There's no point having JSDoc for parameters if it just states the parameter name, I feel?

Reply:
👍